api/src/wfl/server.clj

```diff
@@ -76,9 +77,21 @@
       wrap-internal-error
       (wrap-json-response {:pretty true})))
 
-(defn notify-watchers [watchers _uuid _exception]
+(defn notify-watchers
+  "Notify `watchers` with available approaches."
+  [watchers uuid exception]
+  ;; FIXME: add permission checks for slack-channel-watchers
   {:pre [(some? watchers)]}
-  (log/info (str/join " " ["notifying: " watchers])))
+  (let [slack-msg              (format
+                                (str/join " "
+                                          ["NOTE: WorkFlow Launcher failed to update"
+                                           "a workload %s you are watching! Details"
+                                           "shown below: \n %s"]) uuid exception)
+        slack-channel-watcher? (fn [watcher]
+                                 (not (util/email-address? watcher)))
```

Comment:
```suggestion
        slack-channel-watcher? (complement util/email-address?)
```

Reply:
I think that ship has sailed and would be more complicated to implement. We could make a breaking change of course!

Reply:
Who uses the email notification now?